build: post issues from local roachtests

tbg opened the pull request:
This wasn't happening due to some missing env vars. I noticed this since
there are many failures of acceptance/version-upgrade not reflected via
created issues (I get email notifications from TC).

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/43956)
<!-- Reviewable:end -->

tbg committed:
build: post issues from local roachtests

This wasn't happening due to some missing env vars. I noticed this since
there are many failures of acceptance/version-upgrade not reflected via
created issues (I get email notifications from TC).

Release note: None

andreimatei reviewed (commented):
LGTM

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/43956#-:-LyZukdc-HBI9C9CEPgO:bb6ywmx)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @andreimatei)



<!-- Sent from Reviewable.io -->

tbg commented:
bors r=andreimatei

Failed on two flakes that have issues filed.
